benchflow-test-manager/application/src/test/java/cloud/benchflow/testmanager/helpers/TestBundle.java

```diff
@@ -29,32 +33,25 @@
    */
   public static File getValidTestBundleFile(TemporaryFolder temporaryFolder) throws IOException {
 
-    File tempDir = temporaryFolder.newFolder();
+    // setup the bundle contents
+    ZipEntrySource[] addedEntries = new ZipEntrySource[] {
+        new FileSource(TestFiles.getTestLoadFile().getName(), TestFiles.getTestLoadFile()),
+        new FileSource(TestFiles.getTestDeploymentDescriptorFile().getName(),
+            TestFiles.getTestDeploymentDescriptorFile()),
+        new FileSource("models/" + TestFiles.getModelsFolderFile().listFiles()[0].getName(),
```

Comment:
@jeppe-style you might want to update this so that accessing to the single model file is not hardcoded.